network/src/commonMain/kotlin/dev/icerock/moko/network/features/TokenFeature.kt

```diff
@@ -33,8 +33,7 @@ class TokenFeature private constructor(
         override fun install(feature: TokenFeature, scope: HttpClient) {
             scope.requestPipeline.intercept(HttpRequestPipeline.State) {
                 feature.tokenProvider.getToken()?.apply {
-                    context.headers.remove(feature.tokenHeaderName)
-                    context.header(feature.tokenHeaderName, this)
+                    context.headers[feature.tokenHeaderName] = this
```

Comment:
why this chaged?

Reply:
Because it replace two operation(maybe more, search->get->delete-->add) to one(rewrite | write) => mini performance)

Reply:
it was added because we have case when multiple tokens was sent

Reply:
https://github.com/icerockdev/moko-network/commit/be5ac65b7cf5430a46cb5998dd324edab37fcbcf
https://github.com/icerockdev/moko-network/issues/17